vendor/github.com/mattn/go-shellwords/.travis.yml

```diff
@@ -2,7 +2,6 @@ language: go
 go:
   - tip
 before_install:
-  - go get github.com/axw/gocov/gocov
```

Comment:
trash will only updates based on the version in `trash.conf`. It's v1.0.0 for go-shellwords, so this line should remain (https://github.com/mattn/go-shellwords/blob/v1.0.0/.travis.yml#L5).